Tribler/Core/Modules/restapi/display_endpoint.py

```diff
@@ -21,17 +24,29 @@ def __init__(self, session):
         self.session = session
 
     @staticmethod
-    def return_400(request, message="your request seems to be wrong"):
+    def return_error(request, status_code=http.BAD_REQUEST, message="your request seems to be wrong"):
```

Comment:
@vandenheuvel Please note that this is the default message and every usage in the code has a more specific message. As far as I have seen, other endpoints use a similar message to generalize if they do so.